### ISSUE TITLE
Add interpolation in space without losing interpolatin in time 

### ISSUE THREAD

**Issue by mleprovost** [JuliaDiffEq/DifferentialEquations.jl#395]
Hello,

I am solving a PDE using the ODE solver which gives a solution interpolable in time, basically for free with the option dense=true. My state vector is a quantity f  at different spatial positions. I would like to get an interpolation both in space and time f(X,t). Is there a way to do also the interpolation in space without losing the interpolation in time obtained from the ODE solver?

**Comment by dkarrasch** [comment]
I don't think there can be, because it depends on the spatial arrangement of your ODE nodes. The ODE system/solver does not know or care about it, so you may first need to evaluate your ODE solution at the time of interest, and then interpolate the obtained state vector according to your spatial grid/triangulation/etc using, for instance, `Interpolations.jl`.

**Comment by ChrisRackauckas** [comment]
Yes, the ODE solver's interpolation is very much tied to the time evolution of the system and its calculation. For spatial derivatives you'd want to use a different interpolation method, and I agree with @dkarrasch 's suggestion of Interpolations.jl as a good library for doing so. You can use that in conjunction with the ODE's interpolant to get `u(x,t)` as continuous. If your space is 1-dimensional, I will soon be making public a library DataInterpolations.jl which is just for 1-d interpolations with certain smoothing methods and that could be used here too.
